ovs/lib/alba.py

```diff
@@ -405,7 +412,8 @@ def _add_osds(alba_backend_guid, alba_node_guid, osds, domain, metadata):
                     requested_osd_info['osd_id'] = actual_osd_info['long_id']
                     requested_osd_info[osd_status] = True
 
-        unclaimed = []
+        unclaimed_osds = []
+        failed_claims = []
```

Comment:
Same comment here
Use `unclaimed_osds` and `failure_osds` 
or
`unclaimed` and `failures`